mappings/net/minecraft/block/entity/DecoratedPotBlockEntity.mapping

```diff
@@ -6,13 +6,11 @@ CLASS net/minecraft/unmapped/C_snnddtnv net/minecraft/block/entity/DecoratedPotB
 	FIELD f_ntlmymra sherds Lnet/minecraft/unmapped/C_cpmlbyix;
 	FIELD f_skwmykui lootTableSeed J
 	FIELD f_uumxnicq lastWobbleType Lnet/minecraft/unmapped/C_snnddtnv$C_qjewosgd;
-	FIELD f_yprzpvoh POT_WOBBLES_EVENT I
-	METHOD m_faozmaju wobble (Lnet/minecraft/unmapped/C_snnddtnv$C_qjewosgd;)V
+	METHOD m_faozmaju (Lnet/minecraft/unmapped/C_snnddtnv$C_qjewosgd;)V
 		ARG 1 wobbleType
 	METHOD m_lfhantja asStack ()Lnet/minecraft/unmapped/C_sddaxwyk;
 	METHOD m_mlqltvsn getOrientation ()Lnet/minecraft/unmapped/C_xpuuihxf;
-	METHOD m_rimhwmec getSherds ()Lnet/minecraft/unmapped/C_cpmlbyix;
 	METHOD m_xoyqhuon readNbt (Lnet/minecraft/unmapped/C_sddaxwyk;)V
 		ARG 1 stack
 	CLASS C_qjewosgd WobbleType
-		FIELD f_dwazkvpc duration I
+		FIELD f_dwazkvpc type I
```

Comment:
why? mojang names this `duration`, the usage (DPBERenderer l83) uses it to determine wobble duration, yarn calls this `lengthInTicks`

Reply:
Merging error on my end, fixed!